master/buildbot/test/unit/test_www_avatar.py

```diff
@@ -42,6 +42,15 @@ def test_gravatar(self):
                                     'def654fccc4a4d8?s=32&d=retro'))
 
     @defer.inlineCallbacks
+    def test_github(self):
+        master = self.make_master(url='http://a/b/', auth=auth.NoAuth(), avatar_methods=[avatar.AvatarGitHub()])
+        rsrc = avatar.AvatarResource(master)
+        rsrc.reconfigResource(master.config)
+
+        res = yield self.render_resource(rsrc, '/?email=octocat@github.com')
+        self.assertEqual(res, dict(redirected='https://avatars.githubusercontent.com/u/583231?v=3'))
```

Comment:
looks like this unit test access the network, which we don't do, as this is unreliable, and leads to false positives